src/ClientInterface.php

```diff
@@ -125,6 +125,7 @@
  * @method int               zadd(string $key, array $membersAndScoresDictionary)
  * @method int               zcard(string $key)
  * @method string            zcount(string $key, int|string $min, int|string $max)
+ * @method array             zdiff(int $numkeys, array $keys, bool $withScores = false)
```

Comment:
Perhaps we can remove `numkeys` param from definition and calculate it from length of `keys` when necessary.

Reply:
Yeah, good idea. Let's do that.